eidpdata/src/main/java/uk/ac/ucl/eidp/data/jaxb/StatementGenerator.java

```diff
@@ -35,14 +20,15 @@
 
 /**
  *
- * @author David Guzman <d.guzman at ucl.ac.uk>
+ * @author David Guzman {@literal d.guzman at ucl.ac.uk}
  */
 public class StatementGenerator {
 
   private final XMLInputFactory xif = XMLInputFactory.newFactory();
   private final String DATASET_TAG = "dataset";
   private String SQL_DIALECT = "";
   private String DOT_SEP = "\\.";
+  private final int VARCHAR_LIMIT = 254;
```

Comment:
\[Checkstyle\] WARNING: Abbreviation in name 'VARCHAR\_LIMIT' must contain no more than '1' capital letters\.